packages/core/test/create-app.js

```diff
@@ -320,111 +320,130 @@ describe('create-app', () => {
   });
 
   describe('HTTP after middleware for auth refresh', () => {
-    it('should be applied to OAuth2 refresh app on shorthand requests', (done) => {
-      const oauth2AppDefinition = dataTools.deepCopy(appDefinition);
-      oauth2AppDefinition.authentication = {
-        type: 'oauth2',
-        test: {},
-        oauth2Config: {
-          authorizeUrl: {}, // stub, not needed for this test
-          getAccessToken: {}, // stub, not needed for this test
-          autoRefresh: true,
-        },
-      };
-      const oauth2App = createApp(oauth2AppDefinition);
+    // general purpose response tester
+    const testResponse = (appDef, useShorthand, responseVerifier, done) => {
```

Comment:
Ah, totally fair. I was having some trouble with the tests (the lack of a final `.catch(done)` was causing tests to time out when assertion errors weren't handled), so in the moment, I was just trying to tweak them as little as possible. But you're right, a bit of modernization would probably do us good here since we're in there anyway. 

Great suggestion!